### PR TITLE
Pub status working

### PR DESCRIPTION
**Pull request by DevinSmithWork**
Revised logic for determining "peer reviewed" boolean.

### CODE REVIEW

File: transform.rb [hunk 1]
```diff
@@ -339,8 +348,10 @@ def elementsToJSON(oldData, elemPubID, submitterEmail, metaHash, ark, feedFile)
   # Object type, flags, status, etc.
   elementsPubType = metaHash.delete('object.type') || raise("missing object.type")
   elementsPubStatus = metaHash['publication-status'] || nil
-  data[:isPeerReviewed] = getDefaultPeerReview(elementsPubType,elementsPubStatus)
+  elementsIsReviewed = metaHash['is-reviewed'] || nil
```
Comment (myucekul) [inline transform.rb:351]: Is the custom field coming as 'c-is-reviewed'?

Comment (DevinSmithWork) [reply transform.rb:351]: Yes, via the crosswalk. (Other fields using the c-XYZ have it removed similarly)

Comment (myucekul) [reply transform.rb:351]: Thank you. Also, I see the pattern in the code is to delete the key from hash as it reads the value. I wonder if we should stick to the pattern. (metaHash.delete)

Comment (DevinSmithWork) [reply transform.rb:351]: That's a good idea, I'll revise

**Comment by DevinSmithWork** [comment]
- Requested changes made on transform.rb
- Rest.rb : Added a a commented section with the changes for allowing non-pdf/word deposits on mainfiles.
- Minor commenting cleanups

**Pull request merged**